src/components/Viewer/Viewer.js

```diff
@@ -96,6 +118,30 @@ const Viewer = () => {
                         </MenuItem>
                     ))}
                 </Select>
+                <Box
+                    className={classes.containerCards}
+                >
+                    {selection && Array.isArray(json[selection]) && json[selection].map((card, index) => (
+                        <Card
+                            key={index}
+                            className={classes.jsonCard}
+                            variant='outlined'
+                        >
+                            {console.log(index)}
+                            {console.log(card)}
```

Comment:
{console.log(index)} and {console.log(card)} are not needed, but they are good for info